addons/Popochiu/Engine/Objects/Room/PopochiuRoom.gd

```diff
@@ -253,6 +243,22 @@ func set_active_walkable_area(walkable_area_name: String) -> void:
 		[script_name, walkable_area_name])
 
 
+func get_characters() -> Array:
+	var characters := []
+	
+	for c in $Characters.get_children():
+		if c is PopochiuCharacter:
+			characters.append(c)
+	
+	return characters
+
+
+func clean_characters() -> void:
+	for c in $Characters.get_children():
+		if c is PopochiuCharacter:
+			c.free()
```

Comment:
Isn't it important to use `queue_free()` here? What if you're doing this with the running game for debug?